hazelcast-spring/pom.xml

```diff
@@ -30,7 +30,7 @@
     </parent>
 
     <properties>
-        <org.springframework.version>3.1.0.RELEASE</org.springframework.version>
+        <org.springframework.version>4.0.2.RELEASE</org.springframework.version>
```

Comment:
JCacheCacheManager is present in 3.2.0 at least. However Spring 3.2.0 depends on JCache 0.5.
Spring 4.0.2 both have JCacheCacheManager and depends on JCache 1.0.0. This extension works under Spring 4 and JCache 1.0. It may work with Spring 3.x and JCache 0.5 but I could not test it yet.
Rest of the Spring module should work fine with Spring 3.x